clearnode/rpc_router_private.go

```diff
@@ -833,3 +837,14 @@ func verifySigner(rpc *RPCMessage, channelWallet string) error {
 	}
 	return nil
 }
+
+func ensureWalletHasZeroChannelAllocation(tx *gorm.DB, wallet string) error {
```

Comment:
Also, the function name implies we want to ensure that the wallet has no allocations  or zero allocation only in one channel.
I think the following name provides a better gist of what this function does.

```suggestion
func ensureWalletHasAllAllocationsEmpty(tx *gorm.DB, wallet string) error {
```

Reply:
[3f5595b](https://github.com/erc7824/nitrolite/pull/414/commits/3f5595baa0b08bfcb751aec47ec3245d04930119)